running test_regrid_from_file
Traceback (most recent call last):
  File "/glade/scratch/mpotts/gfortran_7.4.0_openmpi_O/src/addon/ESMPy/src/ESMF/interface/loadESMF.py", line 12, in <module>
    from ESMF.interface.esmfmkfile import ESMFMKFILE as esmfmk
ModuleNotFoundError: No module named 'ESMF.interface.esmfmkfile'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "src/ESMF/test/regrid_from_file/run_regrid_from_file.py", line 6, in <module>
    import ESMF.api.constants as constants
  File "/glade/scratch/mpotts/gfortran_7.4.0_openmpi_O/src/addon/ESMPy/src/ESMF/__init__.py", line 69, in <module>
    from ESMF.api.esmpymanager import *
  File "/glade/scratch/mpotts/gfortran_7.4.0_openmpi_O/src/addon/ESMPy/src/ESMF/api/esmpymanager.py", line 11, in <module>
    from ESMF.interface.cbindings import *
  File "/glade/scratch/mpotts/gfortran_7.4.0_openmpi_O/src/addon/ESMPy/src/ESMF/interface/cbindings.py", line 13, in <module>
    from ESMF.interface.loadESMF import _ESMF
  File "/glade/scratch/mpotts/gfortran_7.4.0_openmpi_O/src/addon/ESMPy/src/ESMF/interface/loadESMF.py", line 14, in <module>
    raise ImportError('The ESMFMKFILE cannot be found!')
ImportError: The ESMFMKFILE cannot be found!
Traceback (most recent call last):
  File "setup.py", line 273, in <module>
    'test_regrid_from_file_parallel': TestRegridFromFileParallelCommand})
  File "/glade/u/apps/ch/opt/python/3.6.8/gnu/8.3.0/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/glade/u/apps/ch/opt/python/3.6.8/gnu/8.3.0/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/glade/u/apps/ch/opt/python/3.6.8/gnu/8.3.0/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "setup.py", line 192, in run
    subprocess.check_call(cmd)
  File "/glade/u/apps/ch/opt/python/3.6.8/gnu/8.3.0/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/glade/u/apps/ch/opt/python/3.6.8/gnu/8.3.0/bin/python3', 'src/ESMF/test/regrid_from_file/run_regrid_from_file.py']' returned non-zero exit status 1.
